docs/understanding-airbyte/high-level-view.md

```diff
@@ -11,16 +11,31 @@ The platform provides all the horizontal services required to configure and run
 Connectors are independent modules which push/pull data to/from sources and destinations. Connectors are built in accordance with the [Airbyte Specification](./airbyte-protocol.md), which describes the interface with which data can be moved between a source and a destination using Airbyte. Connectors are packaged as Docker images, which allows total flexibility over the technologies used to implement them. 
 
 A more concrete diagram can be seen below:
-
-![3.048-Kilometer view](../.gitbook/assets/understanding_airbyte_high_level_architecture.png)
-
-* `UI`: An easy-to-use graphical interface for interacting with the Airbyte API.
+```mermaid
+---
+title: Architecture Overview
+config:
+  theme: neutral
+---
+flowchart LR
+    W[fa:fa-display Airbyte Web UI]
+    S[fa:fa-server Airbyte Server]
+    D[(fa:fa-table Config & Jobs)]
+    T(fa:fa-calendar Temporal)
+    W2[1..n Airbyte Workers]
+    W -->|sends API requests| S
+    S -->|store data| D
+    S -->|create workflow| T
+    T -->|launch task| W2
+    W2 -->|return job| T
+    W2 -->|launches| Source
+    W2 -->|launches| Destination
+```
+
+* `Airbyte Web UI`: An easy-to-use graphical interface for interacting with the Airbyte API.
 * `WebApp Server`: Handles connection between UI and API.
-* `Config Store`: Stores all the connections information \(credentials, frequency...\).
-* `Scheduler Store`: Stores statuses and job information for the scheduler bookkeeping.
-* `Config API`: Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
-* `Scheduler`: The scheduler takes work requests from the API and sends them to the Temporal service to parallelize. It is responsible for tracking success/failure and for triggering syncs based on the configured frequency.
-* `Temporal Service`: Manages the task queue and workflows for the Scheduler. 
+* `Database Config & Jobs`: Stores all the connections information \(credentials, frequency...\).
+* `Airbyte API`: Airbyte's main control plane. All operations in Airbyte such as creating sources, destinations, connections, managing configurations, etc.. are configured and invoked from the API.
```

Comment:
I think this should be called the airbyte server/config API. We are generally trying to use Airbyte API to mean the stable public api.